ATM.py

```diff
@@ -0,0 +1,112 @@
+"""The script simulates ATM.
+
+The script simulates ATM that preforms the following:
+balance check, withdraw cash, deposit cash and
+password change.
+"""
+import os
+import sys
+
+
+BALANCE = 1
+PASSWORD = 0
+ID_INDEX = 0
+PATH_TO_FILE = 1
+INDEX_START_OF_PASSWORD_AND_BALANCE = 1
+
+
+def file_to_dic(filename):
+    """Insert the ATM  data from the file into the dictionary
+
+    Args:
+        filename (str): file name ending with '.txt'
+
+    Returns:
+        dictionary: key=id and value=[password, balance]
+    """
+    atm = {}
+    with open(filename, 'r') as file:
+        for line in file:
+            details = line.strip().split(' ')
+            atm[details[ID_INDEX]] = details[INDEX_START_OF_PASSWORD_AND_BALANCE:]
```

Comment:
Saving the data as list isn't a good practice.
What happen if you change the way the data look? you need to change everywhere you use it the indexes of each part.
Instead the only place that should be effect is the read file function